components/claim-mgt/org.wso2.carbon.identity.claim.metadata.mgt/pom.xml

```diff
@@ -132,6 +136,7 @@
                             !org.wso2.carbon.identity.claim.metadata.mgt.internal,
                             org.wso2.carbon.identity.claim.metadata.mgt.*;version="${carbon.identity.package.export.version}"
                         </Export-Package>
+                        <DynamicImport-Package>*</DynamicImport-Package>
```

Comment:
what is the reason for using DynamicImport-Package? can't we add the required imports under import-package section?